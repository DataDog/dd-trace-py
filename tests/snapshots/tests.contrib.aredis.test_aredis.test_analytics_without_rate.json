[[
  {
    "name": "redis.command",
    "service": "redis",
    "resource": "GET cheese",
    "trace_id": 0,
    "span_id": 1,
    "parent_id": 0,
    "type": "redis",
    "meta": {
      "_dd.base_service": "",
      "_dd.p.dm": "-0",
      "component": "aredis",
      "db.system": "redis",
      "language": "python",
      "out.host": "localhost",
      "redis.raw_command": "GET cheese",
      "runtime-id": "2bc441c84b6b45fc9d6f6fdef59a24f3",
      "span.kind": "client"
    },
    "metrics": {
      "_dd.measured": 1,
      "_dd.top_level": 1,
      "_dd.tracer_kr": 1.0,
      "_dd1.sr.eausr": 1.0,
      "_sampling_priority_v1": 1,
      "db.row_count": 0,
      "network.destination.port": 6379,
      "out.redis_db": 0,
      "process_id": 23297,
      "redis.args_length": 2
    },
    "duration": 839333,
    "start": 1692706868067041590
  }]]
