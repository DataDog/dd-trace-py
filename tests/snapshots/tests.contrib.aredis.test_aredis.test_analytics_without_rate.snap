[[{"name" "redis.command"
   "service" "redis"
   "resource" "GET cheese"
   "type" "redis"
   "error" 0
   "span_id" 0
   "trace_id" 0
   "parent_id" nil
   "start" 1632399428628908532
   "duration" 1261781
   "meta" {"runtime-id" "1853cd9b851147c591ceb9f8865f862c"
           "redis.raw_command" "GET cheese"
           "out.host" "localhost"}
   "metrics" {"_dd.agent_psr" 1.0
              "_dd.measured" 1
              "_dd1.sr.eausr" 1.0
              "_sampling_priority_v1" 1
              "system.pid" 733
              "redis.args_length" 2
              "_dd.tracer_kr" 1.0
              "out.redis_db" 0
              "out.port" 6379}}]]
